### PR TITLE
Building With Active Record

### PR DESCRIPTION
**Pull request by Anvi98**
In this feature branch, We worked with Ruby on Rails To create three models: 
- User
- Post
- Comment
We then used association, migration, and validation techniques to link each model and ensure the correctness of the data inserted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,82 @@
...
+
+clone the project with `git@github.com:arthurborgesdev/micro-reddit.git`
+
+run the migrations with `rails db:migrate`
+
+### Usage
+
+Open a Terminal and run `rails c` to open the console for Rails.
+
+Create the Users, Posts and Comments using the model methods.
```
Comment (enaburekhan) [inline README.md:45]: Make the usage more descriptive by adding a step-by-step process on how to create a `user`, `post`, and `comment`.
E.g to  create a `user`. We can have a step similar to this:
user = User.create(username: "example", email: "example@gmail.com")
Do the same for `post` and `comment`.

**Review by enaburekhan — CHANGES_REQUESTED** [review]
# Status: Changes required:wrench::hammer::full_moon:
**Hi team!
Great job so far:clap::clap:.**

**Major highlights**

- The PR is very descriptive
- The codebase is neat, easy to read and understand.

However, we still need to make a few changes to get the project approved:muscle:.

**To improve**

- [x] App behavior

- A comment can be created ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/creatingComment.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails)). Currently, I can't create a comment. Re-check the model configurations for `comment`.  Then add a similar step to create `comment` just like the step I suggested to create a `user` in the readme. As shown in the PR below.

- [x] Comment

- validations are present (`body` and `title` in automated tests but can be anything else) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/postValidation.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails)). Validations are currently missing. Kindly add them.

- [x] Readme

- As already highlighted in the PR below. Kindly make the suggested changes in the `Usage` section. This will make the readme to be more descriptive.

Make the above-required changes and when you are ready, request another review.

Do have questions/comments? kindly drop them :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy hacking:blush:
Eric Enaburekhan.

File: README.md (hunk 1 above)
Comment (arthurborgesdev) [reply README.md:45]: Hello @enaburekhan, we did the changes that you requested, and now you and the following TSE will be going to succeed in creating the comment with the instructions that we added in README.

**Review by cyonii — APPROVED** [review]
## STATUS: APPROVED ✅

Hello team, well-done implementing all the requested changes. You can now merge this PR and carry on. Congrats, see you at the top.

> #### <br>You can ping me on Slack @[CY Kalu](https://microverse-students.slack.com/team/U01494U96FR) to draw my attention in case I do not respond quickly on here. I'm always available and will be glad to help.<br><br>

**Pull request merged**